starknet_py/common.py

```diff
@@ -41,23 +31,6 @@ def int_from_hex(number: Union[str, int]) -> int:
     return number if isinstance(number, int) else int(number, 16)
 
 
```

Comment:
Don't we want maintain backward compatibility with warnings?

Reply:
Where exactly?